data-reconciliation-app/demo/scripts/01-data_ingest_demo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+sudo rm -rf ./demo/Reports/*.json
+VARS_PATH="./demo/scripts/env_vars.sh"
+. "$VARS_PATH"
+
+cd "${certs}"
+
+printf "Member1 trying to ingest data \n" 
+check_eq "Member1 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member0) -H "Content-Type: application/json" --data-binary "@../../demo/data/member1_demo_pt1.json" $only_status_code)"
+addCheckpoint "✅ Member1 successfully ingested data"
```

Comment:
nit:

We are referencing `Member1` and `Member2` but using the certificates for `member0` and `member1` respectively.
For one who is not used to our flow, this may be confusing when trying to debug.

IMHO it would be worthy referencing the members as per they certificate (so Member0 will be the one specified by member0 certificate and so on)

Reply:
I will change it to Member 0, 1 and 2 then.